Misc/NEWS.d/next/IDLE/2019-05-05-16-27-53.bpo-13102.AGNWYJ.rst

```diff
@@ -1 +1 @@
-When saving a file, call os.flush() so bits are flushed to e.g. USB drive.
+When saving a file, call os.fsync() so bits are flushed to e.g. USB drive.
```

Comment:
We missed the type in the file title.   13102 in the filename was the original pr #.  The bpo in the PR, 36807,  is correct and should have been the bpo # in the filename.  I will fix this in the 3.8 and 3.7 changelogs.

Reply:
Thanks!